syntax = "proto3";

message Null {}

message Time_Calculation {
    string name = 1;
    string description = 2;
    uint32 time_period = 3;
}

message Calc_Result {
    string expected_dateline = 1;
}

service TestService {
    rpc Health(Null) returns (Null);
    rpc AddTimeCalculation(Time_Calculation) returns (Calc_Result);
}